command/kv_get.go

```diff
@@ -81,7 +81,7 @@ KV Get Options:
 
 func (c *KVGetCommand) Run(args []string) int {
 	cmdFlags := flag.NewFlagSet("get", flag.ContinueOnError)
-	cmdFlags.Usage = func() { c.Ui.Output(c.Help()) }
```

Comment:
This looks like an unrelated change, but if this is implied by not setting `.Usage` then makes sense and is fine.